eservice/lib/libpdo_enclave/contract_state.cpp

```diff
@@ -93,47 +95,53 @@ void ContractState::Unpack(const ByteArray& state_encryption_key_,
         if (pvalue != NULL && pvalue[0] != '\0')
         {
             state_hash_ = base64_decode(pvalue);
-            kv_ = new pdo::state::Interpreter_KV(state_hash_, state_encryption_key_);
+            state_ = new pdo::state::Interpreter_KV(state_hash_, state_encryption_key_);
```

Comment:
is there a corresponding delete?

Reply:
cleaned up. though this would be far more robust if we could call a separate method to initialize a kv store with a particular block